colors/yowish.vim

```diff
@@ -77,7 +77,7 @@ call s:Hi('Question'     , 'NONE'                   , s:color.lightGreen      ,
 call s:Hi('Search'       , s:color.yellow           , s:color.backgroundDark  , 'bold')
 call s:Hi('SpecialKey'   , 'NONE'                   , s:color.backgroundLight , 'NONE')
 call s:Hi('Special'      , 'NONE'                   , s:color.textLight       , 'NONE')
-call s:Hi('SpellBad'     , 'NONE'                   , 'NONE'                  , 'undercurl')
+call s:Hi('SpellBad'     , 'NONE'                   , s:color.red             , 'undercurl')
```

Comment:
No a fan here either, but if you need it please make it also optional via an option as explained in my previous comment.